qemu/tests/snp_basic_config.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from avocado.utils import process
+from virttest import data_dir as virttest_data_dir
+from virttest import error_context
+from virttest.utils_misc import verify_dmesg
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu snp basic test on Milan and above host:
+    1. Check host snp capability
+    2. Boot snp VM
+    3. Verify snp enabled in guest
+    4. Check snp qmp cmd and policy
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    error_context.context("Start sev-snp test", test.log.info)
+    timeout = params.get_numeric("login_timeout", 240)
+
+    snp_module_path = params["snp_module_path"]
+    if os.path.exists(snp_module_path):
+        with open(snp_module_path) as f:
+            output = f.read().strip()
+        f.close()
+        if output not in params.objects("module_status"):
+            test.cancel("Host sev-snp support check fail.")
+    else:
+        test.cancel("Host sev-snp support check fail.")
+    singlesocket_host_check = params.get("singlesocket_check_cmd")
+    if singlesocket_host_check:
+        if process.system(singlesocket_host_check, ignore_status=True, shell=True) != 1:
```

Comment:
Interesting here, I just found that you want to check if the returned value is 1, actually it's not a check command 'singlesocket_check_cmd', it's a get command, e.g. 'socket_count_cmd', then you get the returned value and check if it is 1. Here we cannot use system, because system returns the exit code of the running command, we can use getoutput, note that getoutput returns a string, so we need to compare two strings

Reply:
updated to getoutput, and change to int then compare. 
 (3/3) Host_RHEL.m9.u6.v0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9..0.x86_64.io-github-autotest-qemu.snp_basic_config.policy_singlesocket.q35: CANCEL: Host cpu has more than 1 socket, skip the case. (3.10 s)